packages/babel-preset/src/transformPlugin.ts

```diff
@@ -62,6 +77,90 @@ function getCalleeFunctionKind(
   return null;
 }
 
+/**
+ * Gets the id of the parent variable declarator
+ */
+function getParentDeclaratorId(path: NodePath<t.CallExpression> | NodePath<t.MemberExpression>): string {
+  const declarator = path.findParent(p => p.isVariableDeclarator());
+  if (declarator?.isVariableDeclarator()) {
+    const id = declarator.get('id');
+    if (id.isIdentifier()) {
+      return id.node.name;
+    }
+  }
+
+  return 'unknownHook';
```

Comment:
```suggestion
  return 'unknownId';
```

Should it also contain some `id` as a suffix, i.e. `return 'unknownId' + i++`?

Reply:
I haven't really found a real use case where this happens, so I'm also not sure if you'd need to know how many unknowns there are 🤔

Reply:
Sounds fair to me